.github/copilot-instructions.md

```
@@ -2,520 +1,80 @@
+2. **File-type specific guidelines** - Apply based on glob pattern and type
+3. **General guidelines** - Apply as baseline for all files
 
-## Private functions
+### Core Project Guidelines
 
-Private functions (also known as helper functions) should always have its
-separate script file and the function name as the file name with the .ps1
-extension. These files shall always be placed in the folder source/Private.
-This also applies to functions that are only used within a single public
-command.
+- [SqlServerDsc Guidelines](instructions/SqlServerDsc-guidelines.instructions.md) (`**/*.psm1,**/*.psd1,**/*.ps1`)
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 80; Actual: 114

(MD013, line-length)